static/app/views/settings/project/projectFilters/projectFiltersChart.tsx

```diff
@@ -16,44 +19,50 @@ type Props = {
   project: Project;
 };
 
+const STAT_OPS = {
+  'browser-extensions': theme.gray200,
+  cors: theme.yellow300,
+  'error-message': theme.purple300,
+  'discarded-hash': theme.gray200,
+  'invalid-csp': theme.blue300,
+  'ip-address': theme.red200,
+  'legacy-browsers': theme.gray200,
+  localhost: theme.blue300,
+  'release-version': theme.purple200,
+  'web-crawlers': theme.red300,
+  'filtered-transaction': theme.yellow400,
+  'react-hydration-errors': theme.outcome.filtered,
+  'chunk-load-error': theme.outcome.filtered,
+};
+
 function formatData(rawData: UsageSeries | undefined) {
   if (!rawData || !rawData.groups?.length) {
     return [];
   }
 
   const formattedData = rawData.groups
-    .filter(group => STAT_OPS[group.by.reason as keyof typeof STAT_OPS])
     .map(group => {
-      const {title, color} = STAT_OPS[group.by.reason as keyof typeof STAT_OPS];
+      const reason = group.by.reason;
+
+      if (!defined(reason)) {
+        return undefined;
+      }
+
       return {
-        seriesName: title,
-        color,
+        seriesName: startCase(String(reason)),
```

Comment:
Maybe we should pass the name through `t()` as well?

Reply:
it is a good idea, but we usually only use `t()` for content defined in the frontend only and the reason is coming from the backend. Does that make sense?

Reply:
Yes, I guess it makes sense and it is now potentially broader than just the keys to the `STAT_OPS` map.